spring-cloud-azure-autoconfigure/src/main/resources/ApplicationInsights.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings" schemaVersion="2014-05-30">
+    <!-- The key from the portal: -->
```

Comment:
I don't see any useful information in this xml file. Is this file required?

Reply:
From AI SDK will use this, and only fill the Key when release.